src/loki/route_action.cc

```diff
@@ -114,7 +114,9 @@ void loki_worker_t::route(Api& request) {
         }
       }
     }
-  } catch (const std::exception&) { throw valhalla_exception_t{171}; }
+  } catch (const valhalla_exception_t& e) { throw e; } catch (const std::exception&) {
```

Comment:
terrible formatting.. basically re-throwing valhalla_exception_t here which will now be thrown in `GetGraphTile`